interface/patient_file/encounter/delete_form.php

```diff
@@ -17,6 +17,9 @@
 use OpenEMR\Common\Logging\EventAuditLogger;
 use OpenEMR\Core\Header;
 
+require_once(dirname(__FILE__) . "/../../../library/forms.inc");
```

Comment:
this should go above after line 15
(also ensure there's a new line then between this and the `use` block)

Reply:
fixed.